src/token/done-token-parser.js

```diff
@@ -30,7 +31,15 @@ function parseToken(parser, options, callback) {
           rowCount: rowCountValid ? rowCount : undefined,
           curCmd: curCmd
         });
-      });
+      };
+
+      if (options.tdsVersion < '7_2') {
+        parser.readUInt32LE(next);
+      } else {
+        parser.readBigUInt64LE((rowCount) => {
+          next(JSBI.toNumber(rowCount));
```

Comment:
If more than `Number.MAX_SAFE_INTEGER` (`9007199254740991`) rows are transferred, this conversion might result in an incorrect rowcount. I'm not sure if that's something we have to care about - this limit seems hardly reachable. :shrug: